packages/components/pagination/src/js/pagination.html

```diff
@@ -1,104 +1,89 @@
-<div class="oui-pagination__progress">
-    <oui-dropdown placement="center" arrow>
-        <button type="button"
-            class="oui-button oui-button_ghost"
-            oui-dropdown-trigger>
-            <span ng-bind="$ctrl.currentOffset"></span> -
-            <span ng-bind="$ctrl.getLastItemOffset()"></span>
-        </button>
-        <oui-dropdown-content>
-            <div role="menu" class="oui-pagination-menu">
-                <div class="oui-pagination-menu__group"
-                    ng-bind="::$ctrl.config.translations.resultsPerPage"></div>
-                <div class="oui-pagination-menu__items-list">
-                    <button type="button"
-                        ng-repeat="pageSize in $ctrl.pageSizeList track by $index"
-                        class="oui-pagination-menu__item"
-                        ng-disabled="pageSize === $ctrl.pageSize"
-                        ng-bind="pageSize"
-                        ng-click="$ctrl.onPageSizeChange(pageSize)"></button>
-                </div>
-            </div>
-        </oui-dropdown-content>
-    </oui-dropdown>
-    <span class="oui-pagination__progress-text" ng-bind="$ctrl.translations.ofNResults"></span>
+<div class="oui-pagination-result">
+    <label class="oui-select oui-select_inline">
```

Comment:
Why use a `<label>` here? As you don't set any `for` property. `p`or `span` isn't enough?

Reply:
No, it's not really the same.
When we have a control inside a label, we don't need the `for` attribute because the association is implicit (eg. https://www.w3.org/TR/html401/interact/forms.html#edef-LABEL).

Since the `oui-select` has custom part like the chevron icon, we use this to propagate the states. Even if the interest of this behavior for this component is minimal, it takes all its sense for a component like the `oui-checkbox` and the `oui-radio`.